common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisControlManager.java

```diff
@@ -38,6 +38,9 @@ public class TardisControlManager {
     private int[] coordinateIncrements = new int[]{1, 10, 100, 1000};
     private int cordIncrementIndex = 0;
 
+    private boolean autoLand = false;
+    public void setAutoLand(boolean autoLand) {this.autoLand = autoLand;}
```

Comment:
Can we have a getter for this field too? It may be useful for checking Tardis information, and will be useful in inter-mod compatibility.

Reply:
Done